frontend/packages/ui/src/Layouts/PageLayout.tsx

```diff
@@ -241,26 +241,18 @@ const NAV_WIDTH = 300
 const CONTENT_MAX_WIDTH = 1100
 
 export const PageLayout = styled(Page)`
-  display: grid;
   position: relative;
+  display: flex;
+  flex-direction: column;
   min-height: 100vh;
   z-index: 0;
-  grid-template-columns: 0 100%;
```

Comment:
We should remove implementation of the navigation as well, because with this update it will not work any more. Let's wait until I have a chance to test this, maybe it's just better to add padding-bottom for the size of the footer.

Reply:
Ok, I just let me know if this solution is good, and I'll remove implementation of navigation.